src/components/Shop/Shop.js

```diff
@@ -135,6 +139,23 @@ export const Shop = ({
           }}
         />
       </li>
+      {features.MINING ? (
+        <li>
+          <TierPurchase
+            {...{
+              description:
+                'You can purchase a Smelter to convert ore into ingots, and other useful items.',
+              handleTierPurchase: handleSmelterPurchase,
+              maxedOutPlaceholder: "You've already purchased the smelter.",
```

Comment:
Grammar nit: Change this sentence to end with "!" to be consistent with the other `maxedOutPlaceholder` strings. I could be swayed to change them all to sound less excited, but either way they should be consistent.